qemu/tests/cfg/luks_image_over_qsd.cfg

```diff
@@ -0,0 +1,37 @@
+- luks_image_over_qsd:
+    only Linux
+    type = luks_image_over_qsd
+    sleep_time = 5
+    start_vm = no
+    images += " nbd1"
+#### Declare images attributes
+    image_name_stg1 = images/stg1
+    image_format_stg1 = luks
+    image_size_stg1 = 5G
+    image_secret_stg1 = redhat
+    force_create_image_stg1 = yes
+#### Declare QSD (name), multi QSD separated with blank
+    qsd_namespaces = "qsd1"
+#### Declare specified QSD contains which images
+    qsd_images_qsd1 = "stg1"
+    qsd_daemonize_qsd1 = yes
+#### Run QSD in daemon mode ,default is n
+#### Force create QSD even if find same name QSD,default is yes
+    qsd_force_create_qsd1 = yes
+#### Auto adding --pidfile into QSD command line, default is yes
+    qsd_enable_pidfile_qsd1 = yes
+#### Raw command line of QSD daemon,QSD concat them
+    qsd_image_format = {"driver":"raw","read-only":false,"discard":"unmap"}
+#### Declare one image export with nbd inet
+    qsd_image_export_stg1 = {"type":"nbd"}
+    qsd_image_export_nbd_stg1 = {"type":"inet","port":"9000"}
+#### NBD client based on inet
+    force_create_image_nbd1 = no
+    storage_type_nbd1 = nbd
+    image_format_nbd1 = luks
+    enable_nbd_nbd1 = yes
+    nbd_image_tag = nbd1
+    check_image_nbd1 = no
+    nbd_port_nbd1 = 9000
+    nbd_server_nbd1 = localhost
+    qmp_error_msg = "x-blockdev-amend doesn't support changing the block driver"
```

Comment:
we may use %s to replace x-blockdev-amend here, for in future, there may be a command blockdev-amend to replace x-blockdev-amend, so in the py file, we can use err_msg = params['qmp_error_msg'] % blockdev_amend_cmd, where blockdev_amend_cmd can be assigned according to different qemu versions.

But yours is still OK for me, we can update this in future if needed

Reply:
Agree with Zhenchao, we can make changes in future. 

Just a suggestion, we'd better use params to get the qmp command instead of hard code. If blockdev-amend replaces x-blockdev-amend in the future, we only need to update cfg file. 
e.g.
cfg:
blockdev_amend_cmd = "x-blockdev-amend"
qmp_error_msg = "%s doesn't support changing the block driver"
(or qmp_error_msg = "${blockdev_amend_cmd} doesn't support changing the block driver")

py:
For line 80,  we could use the param to get the blockdev_amend cmd instead of hard code.
blockdev_amend_cmd = params["blockdev_amend_cmd"]
qsd.monitor.cmd(blockdev_amend_cmd, args)

For line 82, qmp_error_msg = params['qmp_error_msg'] % blockdev_amend_cmd
(or qmp_error_msg = params["qmp_error_msg"])